use-cases/housekeeping.md

```diff
@@ -0,0 +1,19 @@
+## Housekeeping
+
+A housekeeping integration pulls live information about the physical state of a space and allows housekeeping staff to update it from the housekeeping software, which pushes data back into Mews.
+
+### Initial configuration
+
+The integration must first retrieve all the spaces the property has configured in Mews using [Get all spaces](../operations/enterprises.md#get-all-spaces). The response contains all relevant information needed to map the space configuration of the property in the external system. 
+
+### Managing spaces 
+
+To ensure information is always synchronized between the two systems, the housekeeping integration monitors and manages any changes to the state of a space. Use the websocket [Space event](../websockets.md#space-event) to receive real-time events whenever state of the space is changed. Once a staff member has cleaned or inspected a room, the new state can be sent to Mews using [Update space state](../operations/enterprises.md#update-space-state).
+
+Information about space blocks can be requested using [Get all resource blocks](../operations/enterprises#get-all-resource-blocks). Housekeeping integrations can also manage blocks through [Add resource block](../operations/enterprises.md#add-resource-block) and [Delete resource blocks](../operations/enterprises.md#delete-resource-blocks).
```

Comment:
wrong link missing .md after enterprises

Reply:
.md not required - https://mews-systems.gitbook.io/connector-api/operations/enterprises#get-all-resource-blocks